packages/datagrid/src/components/default-header-renderer/header-grid.scss

```diff
@@ -0,0 +1,23 @@
+.td-header-component {
+	border-right: #e6e6e6 1px solid;
+
+	.td-header {
+		width: 100%;
+		height: 6.6rem;
```

Comment:
There is an issue here : it is not sync with header height set in component or props.